data/112/587/998/3/1125879983.geojson

```diff
@@ -63,7 +67,7 @@
         }
     ],
     "wof:id":1125879983,
-    "wof:lastmodified":1566638482,
+    "wof:lastmodified":1659047515,
     "wof:name":"shokhaib\u064d",
```

Comment:
Nit: Titlecase names, and preferably ASCII7

Reply:
fixed in [e6a2f2d](https://github.com/whosonfirst-data/whosonfirst-data-admin-sa/pull/19/commits/e6a2f2d9a689c22d23c6ceabb5039d78bcfad61a)